vocode/streaming/telephony/conversation/outbound_call.py

```diff
@@ -144,8 +145,8 @@ async def start(self):
             to_phone=self.to_phone,
             from_phone=self.from_phone,
             record=self.telephony_client.get_telephony_config().record,
-            recording_url=(f"{self.telephony_client.get_telephony_config().recording_url}/{self.conversation_id}"),
-            events_url=self.telephony_client.get_telephony_config().events_url,
+            recording_url=f"{self.telephony_client.get_telephony_config().recording_url}/{self.conversation_id}",
+            events_url=f"{self.telephony_client.get_telephony_config().events_url}/{self.conversation_id}",
```

Comment:
unfortunately, due to the way Twilio handles inbound vs outbound, we wont actually be able to support both this way. we'll still need a generic `/events` endpoint, and we'll need to use the CallSid to lookup the ConversationId